qiskit/opflow/gradients/circuit_gradients/lin_comb.py

```diff
@@ -235,7 +241,7 @@ def get_result(item):
                 result.append(get_result(item))
             return result
 
-    @staticmethod
+    @ staticmethod
```

Comment:
Is `@ staticmethod` a mistake? It occurs three times in this file. But, never elsewhere in qiskit/terra

Reply:
It's a mistake 🙂 my editor sometimes formats weirdly if theres a syntax error before that line and fails to un-format it when the error is fixed!